src/requirements/data/majors/cs.ts

```diff
@@ -119,14 +120,15 @@ const csRequirements: readonly CollegeOrMajorRequirement[] = [
   {
     name: 'Major-approved Elective(s)',
     description:
-      'At least 3 credit hours total. All academic courses count.' +
+      'At least 3 credit hours total. All academic courses count. ' +
       'No PE courses, courses numbered 10xx, and ROTC courses below the 3000-level allowed.',
     source:
       'https://www.cs.cornell.edu/undergrad/rulesandproceduresengineering/choosingyourelectives',
     checker: [
       (course: Course): boolean => {
-        const { catalogNbr } = course;
-        return !(ifCodeMatch(course.subject, 'PE') && ifCodeMatch(catalogNbr, '10**'));
+        if (courseIsSpecial(course)) return false;
```

Comment:
Same here. Why would the checker return false for a special course (i.e. an FWS, course that applies to total academic credits requirement) if all academic courses count?

Reply:
See my comment above^